okhttp/src/test/java/okhttp3/OkHttpClientTest.java

```diff
@@ -50,6 +51,8 @@
 import static org.junit.Assert.fail;
 
 public final class OkHttpClientTest {
+  @Rule public PlatformRule platform = new PlatformRule();
```

Comment:
This could be removed but is generally useful.  I was adding because of the assumption in Platform that the system property indicated bouncycastle was in use, vs registration as preferred provider.

Reply:
Should OkHttpClientTestRule do this for all tests? Having one rule that just sets everything up is easy

Reply:
+1 I think you've suggested before, I just haven't done it myself.